resources/osl_nextcloud.rb

```diff
@@ -181,9 +181,9 @@
     action :nothing
   end
 
-  package 'redis'
+  package osl_redis_pkg()
```

Comment:
This can be shortened to the following: Please fix the other instances of this.
```suggestion
  package osl_redis_pkg
```